bin/certbot-auth.sh.aws

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
```

Comment:
Pls. intent this scripts with tabs, to be consistent with the rest of the code.

(otherwise indenting HEREDOCs doesn't work).

Reply:
Done. My code editor has it set up so tabs and spaces look the same, but I've converted this file to use tabs.